sdks/python/apache_beam/utils/python_callable.py

```diff
@@ -88,6 +88,9 @@ def load_from_script(source):
       if line[0] != ' ':
         if line.startswith('def '):
           name = line[4:line.index('(')].strip()
+        elif line.startswith('class '):
+          name = line[5:line.index('(') if '(' in
```

Comment:
Noting that we are extending the definition of the PythonCallable here, from a "function followed by some statements" to a "class followed by some statements that acts as a callable". This can be quite difficult to follow so please add appropriate documentation.

Reply:
Despite the syntax, in all cases it's "the last thing that is defined." Hopefully the docs help. (And the Python tests should serve as good documentation too.)